### PR TITLE
[ITT-629] Cleanup Java library PreServiceAutomation

### PR DESCRIPTION
**Pull request by AlexandrosMor**
**Description**

- Remove config endpoints
- Remove Resource classes
- Remove other unused methods/classes/constants
- Remove unused config properties

**Tested scenarios**

Update the unit tests

### CODE REVIEW

File: src/main/java/com/adyen/Client.java [hunk 1]
```diff
@@ -196,30 +183,10 @@ public void setEnvironment(Environment environment) {
     public void setEnvironment(Environment environment, String liveEndpointUrlPrefix) {
```
Comment (michaelpaul) [inline src/main/java/com/adyen/Client.java:183]: unused variable `liveEndpointUrlPrefix`?

File: src/test/java/com/adyen/ClientTest.java [hunk 2]
```diff
@@ -56,34 +54,6 @@ public void testConfigLiveNoPrefixCheckoutClient() {
...
-        Assert.assertEquals("https://checkoutcert-live-eu.adyen.com/checkout", client.getConfig().getCheckoutEndpoint());
-        assertCommonEndpoints(client.getConfig());
-    }
-
-    @Test
-    public void testClientCertificateAuth_AU() {
-        Client client = new Client(trustStore, clientKeyStore, clientKeyStorePassword, apiKey, Region.AU);
-
-        Assert.assertEquals(Environment.LIVE, client.getConfig().getEnvironment());
-        Assert.assertEquals("https://checkoutcert-live-au.adyen.com/checkout", client.getConfig().getCheckoutEndpoint());
```
Comment (michaelpaul) [inline src/test/java/com/adyen/ClientTest.java:70]: I would be nice to find another way to test the `Region` functionality still works.

File: src/main/java/com/adyen/Client.java [hunk 3]
```diff
@@ -30,43 +30,14 @@
...
-    public static final String ENDPOINT_TEST = "https://pal-test.adyen.com";
-    public static final String ENDPOINT_LIVE = "https://pal-live.adyen.com";
     public static final String ENDPOINT_CERT_LIVE = "https://palcert-live.adyen.com";
-    public static final String ENDPOINT_LIVE_SUFFIX = "-pal-live.adyenpayments.com";
-    public static final String API_VERSION = "v68";
     public static final String PAYOUT_API_VERSION = "v68";
-    public static final String RECURRING_API_VERSION = "v68";
     public static final String LIB_NAME = "adyen-java-api-library";
     public static final String LIB_VERSION = "20.1.1";
     public static final String CHECKOUT_ENDPOINT_TEST = "https://checkout-test.adyen.com/checkout";
```
Comment (jillingk) [inline src/main/java/com/adyen/Client.java:37]: Ditto.

**Comment by jillingk** [comment]
I think there are still some config objects that you can throw away as well https://github.com/Adyen/adyen-java-api-library/blob/develop/src/main/java/com/adyen/Config.java

File: src/main/java/com/adyen/Client.java [hunk 4]
```diff
@@ -30,43 +30,14 @@
 public class Client {
     private ClientInterface httpClient;
     private Config config;
-    public static final String ENDPOINT_TEST = "https://pal-test.adyen.com";
-    public static final String ENDPOINT_LIVE = "https://pal-live.adyen.com";
     public static final String ENDPOINT_CERT_LIVE = "https://palcert-live.adyen.com";
-    public static final String ENDPOINT_LIVE_SUFFIX = "-pal-live.adyenpayments.com";
-    public static final String API_VERSION = "v68";
     public static final String PAYOUT_API_VERSION = "v68";
```
[the remark this answers is not in the document]
Comment (AlexandrosMor) [reply src/main/java/com/adyen/Client.java]: Payout is removed

File: src/main/java/com/adyen/Client.java (hunk 3 above)
Comment (AlexandrosMor) [reply src/main/java/com/adyen/Client.java:36]: This one is removed already `RECURRING_API_VERSION `

**Comment by AlexandrosMor** [comment]
@jillingk good point on the config object #2759399e772514347a5dda8d3b9cd11bb11fdb44

**Review by jillingk — APPROVED** [review]

**Review by michaelpaul — APPROVED** [review]

**Pull request merged**